sdk/core/System.Memory.Data/src/System.Memory.Data.csproj

```diff
@@ -2,11 +2,12 @@
   <PropertyGroup>
     <Description>Contains the BinaryData type, which is useful for converting between string, streams, JSON, and bytes.</Description>
     <AssemblyTitle>System.Memory.Data</AssemblyTitle>
-    <Version>1.0.0-beta.1</Version>
+    <Version>1.0.0-beta.2</Version>
     <Nullable>enable</Nullable>
     <TargetFrameworks>net461;netstandard2.0;</TargetFrameworks>
     <NoWarn>$(NoWarn);AZC0001;AZC0012</NoWarn>
     <AssemblyOriginatorKeyFile>Open.snk</AssemblyOriginatorKeyFile>
+    <PackageIcon>$(RepoEngPath)/images/DotNetPackageIcon.png</PackageIcon>
```

Comment:
I believe this is a path inside the package so you will need to add it to the package and have a relative path to it from there. 

We should also add some condition at https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Directory.Build.Data.targets#L69 so that we don't include our default icon.

Reply:
Perhaps we use a path to the icon and we simply override that here but default that path to our default icon in the props file.